packages/dso-toolkit/src/components/document-component/document-component.scss

```diff
@@ -1,9 +1,6 @@
-@use "../../variables/units";
-@use "../list";
-
 @use "document-component.variables" as document-component-variables;
 
-dso-responsive-element {
+dso-responsive-element.dso-document-components {
```

Comment:
In de praktijk werd deze markup al voorgeschreven. Met deze change is de intentie van de stijling duidelijker.